content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Istio DNS Certificate Management
+description: Shows how to provision and manage DNS certificates in Istio.
+weight: 10
+keywords: [security,certificate]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+By default, the DNS certificates of Galley and Sidecar
+Injector are provisioned and managed by Citadel, which is a large component
+that maintains its own signing key and also acts as a CA for Istio.
+This task shows how to provision and manage DNS certificates in Istio through
+a lightweight component (called Chiron), which signs certificates
+using Kubernetes CA APIs without maintaining its own private key.
+
+## Before you begin
+
+* Create a Kubernetes cluster with Istio installed and the DNS certificate configuration
+in [`values-istio-dns-cert.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml).
```

Comment:
I don't know how to apply the values-istio-dns-cert.yaml. Can you put the installation instructions here if it's not described in /docs/setup/install?

Reply:
Or whatever instruction that can help me to understand how to apply the values-istio-dns-cert.yaml file :)

Reply:
I think it's hard to follow from a user's perspective. I'm not seeing a clear instruction around using "--values" to apply custom yaml files.
@rcaballeromx Hi Rigs, shall we add a tab in 
https://preliminary.istio.io/docs/setup/install/helm/#option-1-install-with-helm-via-helm-template
for applying custom yaml files in helm? Otherwise I think our users will have difficulties applying the values-istio-dns-cert.yaml to helm. WDYT?

Reply:
In the installation page, can we add a tab called "custom helm", and have:
```
$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
    --values ${CUSTOM_HELM} | kubectl apply -f -
```
and we can ask user to set the following here:
```
CUSTOM_HELM=/install/kubernetes/helm/istio/example-values/values-istio-dns-cert.yaml
```
and point them to the "custom helm" tab?

Reply:
Followed the same approach of Istio CNI: added a "DNS certificates" tab and refer to the tab in the user guide. @rcaballeromx, @rlenglet, @myidpt, please take a look.

Reply:
Thanks Lei. That works, although I preferred to add a tab of "custom helm" to cover all these use cases.